contents/docs/surveys/webhook.mdx

```diff
@@ -8,43 +8,8 @@ availability:
   enterprise: full
 ---
 
-import {ProductVideo} from 'components/ProductVideo'
-export const surveyActionsLight = "https://res.cloudinary.com/dmukukwp6/video/upload/v1711621933/posthog.com/contents/docs/surveys-actions-light.mp4"
-export const surveyActionsDark = "https://res.cloudinary.com/dmukukwp6/video/upload/v1711621933/posthog.com/contents/docs/survey-actions-dark.mp4"
-
 It can be useful to receive notifications whenever a user responds to your survey. For example, messages containing their response can be sent to [Slack](/tutorials/slack-surveys), Discord, Teams, or your own custom destination.
 
-## Using webhooks
-
-For Slack, Discord, and Teams, you can set up and use our webhook. To set this up, follow the below steps:
-
-### Step 1: Set up your PostHog Webhook
-
-First you need to set up an [webhook endpoint](/docs/webhooks) where you'll receive survey responses. You can use your own custom endpoint, or you can follow our guides to set one up for [Slack](/docs/webhooks/slack), [Discord](/docs/webhooks/discord), or [Teams](/docs/webhooks/microsoft-teams).
-
-### Step 2: Create an action for survey responses
-
-Next, set up an [action](/docs/data/actions) to post survey events to our webhook. To do this: 
-
-1. Navigate to the [actions tab in Data Management](https://us.posthog.com/data-management/actions) and click **New action** in the top right.
-2. Select **From event or pageview** when prompted.
-3. Under **Match Group #1**, click **All events** and select `survey sent` as the event name.
-4. Check the box which says `Post to webhook when this action is triggered`.
-5. In the **message format** text field, set the text to `New response for [event.properties.$survey_name] from user [person]: "[event.properties.$survey_response]"`
-
-Save your action and you will now begin receiving notifications whenever users respond to your survey.
-
-<ProductVideo
-    videoLight={surveyActionsLight} 
-    videoDark={surveyActionsDark}
-    alt="How to create an action for survey responses" 
-    classes="rounded"
-/>
-
-## Using realtime destinations
-
-Our new [realtime destinations](/docs/cdp/destinations) are currently in preview. These provide customizable destinations, more formatting options, a revamped configuration UI. You can use them to send [survey](/docs/surveys) responses to Slack or any other destination that supports webhooks.
-
 To do this, start by enabling the **Pipeline destinations 3000** feature preview.
```

Comment:
Realtime destinations are not hidden behind a feature preview anymore.

Reply:
good catch, thanks